app.js

```diff
@@ -1,8 +1,86 @@
 // import functions and grab DOM elements
+const leftHatEl = document.getElementById('hat-left');
+const centerHatEl = document.getElementById('hat-center');
+const rightHatEl = document.getElementById('hat-right');
+
+const leftButtonEl = document.getElementById('button-left');
+const centerButtonEl = document.getElementById('button-center');
+const rightButtonEl = document.getElementById('button-right');
+
+const correctTotals = document.getElementById('wins');
+const attemptTotals = document.getElementById('total');
+const incorrectTotals = document.getElementById('losses');
+
+
 
 // let state
+let wins = 0;
+let attempts = 0;
+
+// console.log(wins, attempts);
 
 // set event listeners 
+
+leftButtonEl.addEventListener('click', () =>{
+    const correctGuess = getRandomHat();
+
+    handleGuess('left', correctGuess);
+});
+
+centerButtonEl.addEventListener('click', () =>{
+    const correctGuess = getRandomHat();
+    handleGuess('center', correctGuess);
+});
+rightButtonEl.addEventListener('click', () =>{
+    const correctGuess = getRandomHat();
+    handleGuess('right', correctGuess);
+});
+// function resetStyles()
+
+function getRandomHat(){
+    const hiddenBall = [
+        'left',
+        'center',
+        'right',
+    ];
+    const index = Math.floor(Math.random() * 3);
+    // console.log('index', index);
+    return hiddenBall[index];
+}
+
+
+
+
+function handleGuess(userGuess, correctGuess){ 
+    leftHatEl.src = './assets/cup.png';
+    centerHatEl.src = './assets/cup.png';
+    rightHatEl.src = './assets/cup.png';  
+    
+    attempts++;
+    
+    const correctImage = document.getElementById(`hat-${correctGuess}`); 
+    correctImage.src = './assets/ball.png';
```

Comment:
do you need this if you have your logic on lines 68-78 setting the .src to ball.png? or maybe do you need that logical if statement if you're changing the src up here? This might be causing a weird bug i'm getting.